samples/tab-sso/nodejs/README.md

```diff
@@ -233,6 +233,4 @@ For more information see the [Code of Conduct FAQ](https://opensource.microsoft.
 contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
 ## Further Reading.
```

Comment:
I think this README is not describing use the sample in Teams Toolkit, but describing use the sample without TTK. Since Teams Toolkit will register Azure AD for the tab app when debuging, and fill in the value back to manifest file, we don't need to ask user to do these manually in prerequsites section.